crates/op-rbuilder/src/builders/flashblocks/payload.rs

```diff
@@ -535,7 +538,7 @@ where
                                 .record(flashblock_build_start_time.elapsed());
                             ctx.metrics
                                 .flashblock_byte_size_histogram
-                                .record(new_payload.block().size() as f64);
+                                .record(calculate_flashblock_byte_size(&fb_payload) as f64);
```

Comment:
there's no fb_payload.size() function? I think there should be an inbuilt function to calculate the byte size

Reply:
size would only calculate stack side of the structure